src/plugins_exts.h

```diff
@@ -130,14 +130,14 @@ void lyplg_ext_instance_substatements_free(struct ly_ctx *ctx, struct lysc_ext_s
  * function can be used to let the compilation to libyang following the standard rules for processing the YANG statements.
  *
  * @param[in] cctx Current compile context.
- * @param[in] p_ext Parsed extension instance data.
+ * @param[in,out] p_ext Parsed extension instance data. Item ::lysc_ext_instance.parsed can be modified.
```

Comment:
Yes, it can, but I would pretend it cannot :) By passing a non-`const` parameter in a function you are saying it can be modified, but the user should definitely not modify this parsed structure and treat it as `const` instead. Yes, strictly speaking, it may be modified but only by our function and only by caching the parsed tree in it. So, in order for the `const` keyword to be useful, it may not always be treated that strictly. Please leave the `const` here and everywhere else.